fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -824,7 +836,8 @@ private void checkGroupExportStatus(boolean s3) throws Exception {
         // Export finished successfully, we should be about to find the "output" part in the message body
         // which includes all the COS objects download urls.
         String body = response.readEntity(String.class);
-        assertTrue(body.contains("output"));
+        JsonObject jsonObject = Json.createReaderFactory(null).createReader(new StringReader(body)).readObject();
```

Comment:
I think this is fine, however there is already a factory in place on 111

Reply:
updated to use the factory constant